config/routes.rb

```diff
@@ -30,6 +30,9 @@
       resources :communities, only: [:index, :show]
       resources :events, only: [:index, :show]
       resources :radios, only: [:index, :show]
+      resource :contacts, only: [:index, :create] do
+        get 'enum', to: 'contacts#enum'
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [SideCI](https://sideci.com) -->